src/index.html

```diff
@@ -1,13 +1,843 @@
+            </div>
+          </div>
+
+          <div class="testimonials__card testimonials__card-yellow">
+            <div class="testimonials__picture">
+              <img
+                src="images/girl-2.png"
+                alt="image"
+                class="testimonials__img testimonials__img-two"
+              >
+
+              <div
+                class="testimonials__block testimonials__block-1 testimonials__block-yellow testimonials__block-yellow--3"
+              >
+              </div>
+
+              <div
+                class="testimonials__block testimonials__block-2 testimonials__block-yellow testimonials__block-yellow--4"
+              >
+              </div>
+            </div>
+
+            <div class="testimonials__card-text">
+              <p
+                class="testimonials__card-quotes testimonials__card-quotes--yellow"
+              >
+                &#8220
+              </p>
+
+              <p class="testimonials__card-text--1">
+                AIR is an exceptional agency that leads with creative talent,
+                first-class account servicing.
+              </p>
+            </div>
+
+            <div class="testimonials__card-text-2">
+              <h3 class="testimonials__card-title">
+                Azadeh Hawkins
+              </h3>
+
+              <p class="testimonials__card-text--2">
+                Hawkins Consulting
+              </p>
+            </div>
+          </div>
+
+          <div class="testimonials__card testimonials__card-red">
+            <div class="testimonials__picture">
+              <img
+                src="images/girl-3.png"
+                alt="image"
+                class="testimonials__img testimonials__img-three"
+              >
+
+              <div
+                class="testimonials__block testimonials__block-1 testimonials__block-red testimonials__block-red--5"
+              >
+              </div>
+
+              <div
+                class="testimonials__block testimonials__block-2 testimonials__block-red testimonials__block-red--6"
+              >
+              </div>
+            </div>
+
+            <div class="testimonials__card-text">
+              <p
+                class="testimonials__card-quotes testimonials__card-quotes--red"
+              >
+                &#8220
+              </p>
+
+              <p class="testimonials__card-text--1">
+                AIR raises the agency bar to stratospheric
+                heights on both creative output and client service.
+              </p>
+            </div>
+
+            <div class="testimonials__card-text-2">
+              <h3 class="testimonials__card-title">
+                Michel Grover
+              </h3>
+
+              <p class="testimonials__card-text--2">
+                Hulu
+              </p>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="vision">
+        <div class="vision__content wrapper__vision">
```

Comment:
dont understand these paddings

Reply:
It likes goot to me, it's convenient for me